.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -47,6 +47,7 @@ Please do not remove items from the checklist
       Link: <!-- example https://github.com/kubernetes-sigs/kueue/releases/tag/v0.1.0 -->
 - [ ] Run the [openvex action](https://github.com/kubernetes-sigs/kueue/actions/workflows/openvex.yaml) to generate openvex data. The action will add the file to the release artifacts.
 - [ ] Run the [SBOM action](https://github.com/kubernetes-sigs/kueue/actions/workflows/sbom.yaml) to generate the SBOM and add it to the release.
+- [ ] Update the value of the key `dependencies.sbom.sbom-file` in `SECURITY-INSIGHTS.yaml` with the SBOM generated in the previous step.
```

Comment:
line 58 already lists other changes to the same file. Let's put it there.

Reply:
Good point. I didn't realize the file is already covered elsewhere 😅